submissions/AbrahamJudah/HTML/styles.css

```diff
@@ -0,0 +1,75 @@
+body {
+    background-color: rgb(255,255,255);
+    margin: 100px ;
+}
+.registration-block {
+    margin: 50px;
+    margin-top: 20px;
+    padding: 20px;
+    width: 450px;
+    height: 750px;
+    border: 5px solid red;
+    box-sizing: border-box;
+    position: absolute;
+    border-radius: 20px;
+    background-color: rgba(169, 169, 169, 0.322);
+}
+.header {
+    text-align: center;
+    font-size: 35px;
+    color: navy;
+    display: inline-block; 
+    margin-right: 100px;
+}
+.registration-wrapper {
+    margin-left: 300px;
```

Comment:
Try `margin:auto` to automatically center the registration wrapper. So you won't have to specify margin-left and margin-right